packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js

```diff
@@ -6,6 +6,7 @@ import { getSortByKey, getSortByExtractedValue, getUniqueEntries } from '@tupaia
 
 import { NO_DATA_AVAILABLE } from '/apiV1/dataBuilders/constants';
 import { transformValue } from 'apiV1/dataBuilders/transform';
```

Comment:
@billli0  Just a clue - I can see that some internal imports in `web-config-server` do not have an `'/` prefix. Perhaps those are messing up with path resolution during builds:

```suggestion
import { transformValue } from '/apiV1/dataBuilders/transform';
```

We use [babel-module-resolved[https://www.npmjs.com/package/babel-plugin-module-resolver) to use aliased imports, just in web-config-server

Reply:
Here comes the interesting thing, it works with 

>import { translateEventEntityIdsToNames } from '/apiV1/dataBuilders/helpers/translateEventEntityIdsToNames';

but not 

> import { translateEventEntityIdsToNames } from '/apiV1/dataBuilders/helpers'

Reply:
This is weird? Perhaps because of other internal imports lying around without a `/` prefix? Or it could be unrelated. Who knows! 🤷